container/jbossweb/pom.xml

```diff
@@ -71,7 +71,7 @@
         <profile>
             <id>AS7</id>
             <activation>
-                <activeByDefault>false</activeByDefault>
+                <activeByDefault>true</activeByDefault>
```

Comment:
setApplicationSessionLifecycleListeners method is not found in JBossWeb 2 so had to compile against 7.  The listener is invoked with that, but it doesn't fix the issue because JBossWeb invokes the listener before the session is removed.